src/index.js

```diff
@@ -494,49 +500,6 @@ const initialConnectedButtons = [
   maliciousPermitIntAddress,
 ];
 
-// Buttons that are available after connecting via Wallet Connect
```

Comment:
since this is now fixed, we can remove this section:

> For Wallet Connect, we only have a subset of functionalities enabled for now: Sends/Signatures but excluding Contract Deployments, as needs some more investigation on how to get the contract address. This can be extended in the future (not currently needed for the purposes of PPOM).

[see ref](https://github.com/MetaMask/test-dapp/pull/284)